gp-includes/advanced-permissions.php

```diff
@@ -49,7 +49,31 @@ function gp_allow_everyone_to_translate( $verdict, $args ) {
 	return $verdict;
 }
 
+/**
+ * Map the translation check to the translation-set
+ *
+ * @param  string|bool $verdict Previous decision whether the user can do this.
+ * @param  array       $args Permission details.
+ * @return string|bool New decision whether the user can do this.
+ */
```

Comment:
* "Maps"
* Full stop is missing
* Missing alignment for @param descriptions
* @param and @return value shouldn't be aligned
* @since tag is missing